### PR TITLE
Change loaders version to ^3.0.0-alpha

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
Updated `@loaders.gl` version to `3.0.0-alpha`

### CODE REVIEW

**Comment by Pessimistress** [comment]
Can you limit the lockfile changes to just loaders.gl related? We should not bump all dependencies in one commit.

**Comment by dryabinin-actionengine** [comment]
> Can you limit the lockfile changes to just loaders.gl related? We should not bump all dependencies in one commit.

Done. Please take a look.

File: examples/layer-browser/package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "start-local-production": "webpack-dev-server --env.local --env.production --progress --hot --open"
   },
   "dependencies": {
-    "@loaders.gl/ply": "^2.3.9",
-    "@loaders.gl/gltf": "^2.3.9",
+    "@loaders.gl/ply": "^3.0.0-alpha",
```
Comment (ibgreen) [inline examples/layer-browser/package.json:12]: @Pessimistress Does this work? I always seem to need to specify the actual alpha version? If it does work that is great.

**Review by ibgreen — APPROVED** [review]

File: examples/layer-browser/package.json (hunk 1 above)
Comment (Pessimistress) [reply examples/layer-browser/package.json:12]: Yes that works. You'll have to specify a version if you want to bump to another alpha release later.

**Pull request merged**